src/manager.cpp

```diff
@@ -616,11 +616,13 @@ int Manager::updateKeywordOnHardware(
     const types::Path& i_fruPath, const nlohmann::json& i_sysCfgJsonObj,
     const types::WriteVpdParams i_paramsToWriteData)
 {
-    (void)i_fruPath;
-    (void)i_sysCfgJsonObj;
-    (void)i_paramsToWriteData;
+    std::shared_ptr<vpd::Parser> l_parserObj =
```

Comment:
vpd is implicit,  although having it does not cause an issue but it is not consistent with rest of the file.

Reply:
done